apps/src/templates/progress/progressHelpers.js

```diff
@@ -178,6 +178,12 @@ export const processedLevel = level => {
     icon: level.icon,
     isUnplugged: level.display_as_unplugged,
     levelNumber: level.kind === LevelKind.unplugged ? undefined : level.title,
+    bubbleText:
+      level.kind === LevelKind.unplugged
+        ? undefined
+        : isSublevel
+        ? level.letter
+        : level.title.toString(),
```

Comment:
Why do we need the toString() here?  Can title be something other than a string?

Reply:
for non-unplugged levels, `title` is actually `levelNumber`, and is a number (from [here](https://github.com/code-dot-org/code-dot-org/blob/e346e1830f1d4b28ccf10b9f34555e3da0aacf51/dashboard%2Fapp%2Fmodels%2Fscript_level.rb#L339)).

Reply:
Oye.  At times like this, I feel like a static-typing curmudgeon.  :-)